packages/tocco-ui-showcase/src/components/StyledShowCaseApp.js

```diff
@@ -0,0 +1,105 @@
+import styled, {injectGlobal} from 'styled-components'
+
+// eslint-disable-next-line chai-friendly/no-unused-expressions
```

Comment:
Please consider every eslint-disable very careful. In this case its a valid expression and no need to disable linting for this line if eslint is configured correctly.

.eslintrc

    "chai-friendly/no-unused-expressions": [2, {"allowTaggedTemplates": true}],

would solve it in this case